### PR TITLE
Add parent_sku field #84

### PR DESCRIPTION
**Pull request by Hypo386**
Added parent_sku field.

e.g.

"parent_sku":["MH01","MH02"]

Added new parent_data data provider Divante\VsbridgeIndexerCatalog\Model\Indexer\DataProvider\Product\ParentData for indices.

### CODE REVIEW

File: src/module-vsbridge-indexer-catalog/Model/Product/ParentResolver.php [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    {
+        $parentSkus = [];
+
+        $this->filterBuilder->setField('entity_id');
+        $this->filterBuilder->setValue($parentIds);
+        $this->filterBuilder->setConditionType('in');
+
+        $this->searchCriteriaBuilder->addFilters([$this->filterBuilder->create()]);
+
+        $searchResult = $this->productRepository->getList($this->searchCriteriaBuilder->create());
```
Comment (afirlejczyk) [inline src/module-vsbridge-indexer-catalog/Model/Product/ParentResolver.php]: The idea of an indexer is to work fast -  we don't use repository in indexer, mostly plain sql.
Please use plain sql to get Parent ids.

File: src/module-vsbridge-indexer-catalog/Model/Indexer/DataProvider/Product/ParentData.php [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function addData(array $indexData, $storeId)
+    {
+        $parentIds = [];
+
+        foreach ($indexData as $productId => $productData) {
```
Comment (afirlejczyk) [inline src/module-vsbridge-indexer-catalog/Model/Indexer/DataProvider/Product/ParentData.php]: As it's not use by default VSF yet, so exporting parent sku - should be optional.
Please add an option to configuration - so user can choose if we should export ParentSku or not - by default set it to NO.

File: src/module-vsbridge-indexer-catalog/Model/Indexer/DataProvider/Product/ParentData.php [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
```
Comment (afirlejczyk) [inline src/module-vsbridge-indexer-catalog/Model/Indexer/DataProvider/Product/ParentData.php]: remove empty line after `foreach` statement

File: src/module-vsbridge-indexer-catalog/Model/Product/ParentResolver.php (hunk 1 above)
Comment (Hypo386) [reply src/module-vsbridge-indexer-catalog/Model/Product/ParentResolver.php]: Fixed

File: src/module-vsbridge-indexer-catalog/Model/Indexer/DataProvider/Product/ParentData.php (hunk 2 above)
Comment (Hypo386) [reply src/module-vsbridge-indexer-catalog/Model/Indexer/DataProvider/Product/ParentData.php]: Fixed

File: src/module-vsbridge-indexer-catalog/Model/Indexer/DataProvider/Product/ParentData.php (hunk 3 above)
Comment (Hypo386) [reply src/module-vsbridge-indexer-catalog/Model/Indexer/DataProvider/Product/ParentData.php]: Fixed

**Comment by afirlejczyk** [comment]
Thanks for your PR.
I resolve conflict with new PR: https://github.com/DivanteLtd/magento2-vsbridge-indexer/pull/257

**Pull request closed without merging**